static/js/spoilers.js

```diff
@@ -33,6 +33,19 @@ function expand_spoiler(spoiler) {
     });
 }
 
+exports.hide_spoilers_in_notification = (content) => {
+    content.find('.spoiler-block').each((i, elem) => {
+        $(elem).find('.spoiler-content').remove();
+        let text = $(elem).find('.spoiler-header').text().trim();
+        if (text.length > 0) {
+            text = `${text} `;
+        }
+        text = `${text}(...)`;
```

Comment:
Its not obvious that the message contains a spoiler when the spoiler doesn't have any header text. Using `(***)` as the placeholder felt... weird, but that's another alternative.

I don't want to add a new string here that needs to be translated.

Reply:
I think the `...` is a good solution but I'd put a space after it?

Reply:
Oh, I see, yeah your code is fine; I just didn't expect the way you wrote the two cases.

Reply:
Haha, I was avoiding writing the `(...)` bit twice. It looks like a readability concern then, so I'll maybe refactor this into two explicit string concatenations in a followup PR